Hadrons/Modules/MDistil/LapEvec.hpp

```diff
@@ -101,64 +91,95 @@ struct LapEvecPar: Serializable {
  
  ******************************************************************************/
 
-template <typename GImpl>
+template <typename FImpl>
```

Comment:
Why does is depends on a fermion implementation? The Laplacian should have to know nothing about fermions

Reply:
I needed that to write the Eigenvectors as std::vector<ColourVectorField>, which is part of Ferm_type_aliases. Also the definition of the 3D laplacian takes ColourVectorField as a template input.